Packages/MIES/MIES_Constants.ipf

```diff
@@ -1718,3 +1718,478 @@ Constant DECIMATION_MINMAX = 0x1
 
 StrConstant DEFAULT_KEY_SEP = ":"
 StrConstant DEFAULT_LIST_SEP = ";"
+
+/// \rst
+///
+/// =====  ======  ===============
+/// Name   Symbol  Numerical value
+/// =====  ======  ===============
+/// yotta    Y        1e24
+/// zetta    Z        1e21
+/// exa      E        1e18
+/// peta     P        1e15
+/// tera     T        1e12
+/// giga     G        1e9
+/// mega     M        1e6
+/// kilo     k        1e3
+/// hecto    h        1e2
+/// deca     da       1e1
+/// deci     d        1e-1
+/// centi    c        1e-2
+/// milli    m        1e-3
+/// micro    mu       1e-6
+/// nano     n        1e-9
+/// pico     p        1e-12
+/// femto    f        1e-15
+/// atto     a        1e-18
+/// zepto    z        1e-21
+/// yocto    y        1e-24
+/// =====  ======  ===============
+///
+/// \endrst
+///
+/// From: 9th edition of the SI Brochure (2019), https://www.bipm.org/en/publications/si-brochure
+StrConstant PREFIX_SHORT_LIST = ";Y;Z;E;P;T;G;M;k;h;da;d;c;m;mu;n;p;f;a;z;y"
+StrConstant PREFIX_LONG_LIST  = "one;yotta;zetta;exa;peta;tera;giga;mega;kilo;hecto;deca;deci;centi;milli;micro;nano;pico;femto;atto;zepto;yocto"
+StrConstant PREFIX_VALUE_LIST = "1;1e24;1e21;1e18;1e15;1e12;1e9;1e6;1e3;1e2;1e1;1e-1;1e-2;1e-3;1e-6;1e-9;1e-12;1e-15;1e-18;1e-21;1e-24"
+
+/// @name Conversion constants for decimal multiples
+///
+/// These must be used only with `*` to avoid confusion, the inverse exists
+/// as well, so this is no limitation.
+///
+/// Example:
+///
+/// \rst
+/// .. code-block:: igorpro
+///
+///    // pA -> A
+///    value_A = value_pA * PICO_TO_ONE
+///
+///    // V -> mV
+///    value_mV = value_V * ONE_TO_MILLI
+/// \endrst
+///
+/// Generated code from GenerateMultiplierConstants().
```

Comment:
it's cool that code was used to generate the constant list.

Reply:
@t-b was code also used to find and replace using the constants or is it necessary to hand check each replacement?

Reply:
This can't be done automatically. I did every replacement by hand, this is also the reason why it is grouped in the commits by constants.

For example the factor 1e-3 is either MILLI_TO_ONE or MICRO_TO_MILLI, it depends on the context which one is correct.